### PR TITLE
Add test for extracting the skills

### LINKED ISSUE #31: Write a test for extraction code

### PR DESCRIPTION
**Pull request by TejasPrabhu**
closes #31

### CODE REVIEW

**Comment by TejasPrabhu** [comment]
closes #33

**Review by kartikrawool — APPROVED** [review]

**Review by crmgogo — APPROVED** [review]
looks good

File: src/scraper.py [hunk 1]
```diff
@@ -197,9 +198,10 @@ def extract_skill(self):
         self.job_data['skills'] = skill_list
 
 
-# jd = JobData(job_title="Software Engineer",
-#              job_location="Raleigh",
-#              distance=200,
-#              company="",
-#              number_jobs=10)
-# jd.scrape_data()
+if __name__ == '__main__':
```
Comment (samarth-p) [inline src/scraper.py:201]: Better to keep this commented out. Otherwise, it will be called everytime we run the tests.

File: src/scraper.py [hunk 2]
```diff
@@ -142,10 +143,10 @@ def linkedin_scraper(self, max_jobs=25):
                     By.CLASS_NAME, 'jobs-search__results-list').find_elements(
                     By.CLASS_NAME, 'job-search-card')
 
-                # if len(job_list) > max_jobs:
```
Comment (samarth-p) [inline src/scraper.py:145]: The commented parts can be removed

File: src/scraper.py (hunk 1 above)
Comment (TejasPrabhu) [reply src/scraper.py:201]: Are we going to call scraper.py for our scraping routine or are we going to create a new driving function for cron job? Something to think about. I'll comment it for now.

File: src/scraper.py (hunk 2 above)
Comment (TejasPrabhu) [reply src/scraper.py:145]: what value do we want to set for max_jobs? 25 seems too low.

**Pull request merged**